tasks/utils.js

```diff
@@ -27,8 +27,20 @@ module.exports.getEnvName = function () {
     return argv.env || 'development';
 };
 
-module.exports.getSigningId = function () {
-    return argv.sign;
+module.exports.getSigningId = function (manifest) {
+    return argv.sign || (manifest.codeSignIdentitiy ? manifest.codeSignIdentitiy.dmg : undefined);
```

Comment:
Yes, there are 3 signs inside the `codeSignIdentitiy`, `dmg`, `MAS` and `MASInstaller`, the `dmg` is the normal app signing.